### ISSUE TITLE
:dir selector not working on polymer 1.11.0

### ISSUE THREAD

**Issue by valdrinkoshi** [Polymer/polymer#4952]
Styling with `:dir` selector doesn't work properly in Safari with polymer 1.x

E.g. consider the following element template:
```html
  <style>
    div {
      color: blue;
    }
    :host(:dir(rtl)) div  {
      color: red;
    }
  </style>
  <div>am i red?</div>
```
The div element will be wrongfully blue when we set `<html dir="rtl">`. This works as expected with polymer 2.x, but not with polymer 1.x
http://jsbin.com/pujiden/1/edit?html,output

**Comment by bicknellr** [comment]
If you replace `:host(:dir(rtl)) div` with `:dir(rtl) div` this seems to work. Maybe `:dir()` isn't beingt rewritten inside `:host()`?

**Comment by valdrinkoshi** [comment]
Unfortunately that's not working in Chrome w/ dom=shady (default in Polymer 1.x), see http://jsbin.com/yosageyiwu/1/edit?html,output
- dom=shadow http://output.jsbin.com/yosageyiwu/1?dom=shadow
- dom=shady http://output.jsbin.com/yosageyiwu/1

**Comment by azakus** [comment]
Found where this is broken, it might take a while 😞

**Comment by azakus** [comment]
Fixed with #4964
